docs/tutorials/reactjs-tutorial.md

```diff
@@ -0,0 +1,273 @@
+---
+Order: 5
+Area: nodejs
+TOCTitle: React Tutorial
+ContentId: 2dd2eeff-2eb3-4a0c-a59d-ea9a0b10c468
+PageTitle: React JavaScript Tutorial in Emacs
+DateApproved: 9/10/2020
+MetaDescription: React JavaScript tutorial showing IntelliSense, debugging, and code navigation support in the Emacs editor.
+---
+# Using React in Emacs
+
+[React](https://facebook.github.io/react/) is a popular JavaScript library developed by Facebook for building web application user interfaces. The Emacs editor supports React.js IntelliSense and code navigation out of the box.
+
+![Welcome to React](images/reactjs/welcome-to-react.png)
+
+## Welcome to React
+
+We'll be using the `create-react-app` [generator](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) for this tutorial. To use the generator as well as run the React application server, you'll need [Node.js](https://nodejs.org/) JavaScript runtime and [npm](https://www.npmjs.com/) (Node.js package manager) installed. npm is included with Node.js which you can download and install from [Node.js downloads](https://nodejs.org/en/download/).
+
+>**Tip**: To test that you have Node.js and npm correctly installed on your machine, you can type `node --version` and `npm --version` in a terminal or command prompt.
+
+You can now create a new React application by typing:
+
+```bash
+npx create-react-app my-app
+```
+
+where `my-app` is the name of the folder for your application. This may take a few minutes to create the React application and install its dependencies.
```

Comment:
```suggestion
where `my-app` is the name of the folder for your React application. It may take a few minutes to create it and to install its dependencies.
```

Grammar